generate.2.sh

```diff
@@ -1,8 +1,6 @@
 #!/bin/bash
 
-BASE=$(pwd)
-
-while getopts g:a:t:p:o:v:m:u: option
+while getopts g:a:t:p:o:v:m:u:w: option
```

Comment:
It was required to move the `requirements.txt` to the root directory of the project because this is the only way Heroku can recognise that the project contains Python dependencies. 
It caused a lot of problem to me while i was trying the deploy the app as i received the error
`Failed to detect app matching ....python.tgz buildpack`